<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpubExportDialog</class>
 <widget class="QDialog" name="EpubExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EPUB Export</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>591</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="exportButton">
      <property name="toolTip">
       <string>Leave colors untouched</string>
      </property>
      <property name="text">
       <string>E&amp;xport</string>
      </property>
      <property name="shortcut">
       <string>Alt+C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="toolTip">
       <string>Leave colors untouched</string>
      </property>
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut">
       <string>Alt+C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="fileOutputLabel">
      <property name="text">
       <string>&amp;Export to File:</string>
      </property>
      <property name="buddy">
       <cstring>fileOutput</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileOutput"/>
    </item>
    <item>
     <widget class="QPushButton" name="fileOutputChange">
      <property name="text">
       <string>C&amp;hange...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="rangeGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Range</string>
   </property>
   <layout class="QGridLayout" name="_3">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="2" column="0">
     <widget class="QRadioButton" name="allPagesRadio">
      <property name="text">
       <string>&amp;All pages</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="intervalPagesRadio">
      <property name="text">
       <string>&amp;Range</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="thisSectionRadio">
      <property name="text">
       <string>Current &amp;section</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="intervalPagesButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="intervalPagesValue">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="thisPageRadio">
      <property name="text">
       <string>&amp;Current page</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Text options</string>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep text color</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep font information</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Image options</string>
   </property>
   <widget class="QCheckBox" name="imageMaxWidthCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Downsize images</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="imageMaxWidthSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> px</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>600</number>
    </property>
   </widget>
   <widget class="QLabel" name="imageMaxWidthLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>40</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximal width</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageMaxWidthThresholdLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>64</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximize if bigger than</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="imageMaxWidthThresholdSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> %</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
